JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java

```diff
@@ -89,6 +89,7 @@ public class QueryClient {
     public static final String NX = PREFIX_DISP + "nx";
     public static final String NY = PREFIX_DISP + "ny";
     public static final String SCOPE = PREFIX_DISP + "Scope";
+    public static final String CITIES_NAMESPACE = "https://www.theworldavatar.com/kg/ontodispersion/CitiesNamespace";
```

Comment:
Changed to https://www.theworldavatar.com/kg/ontodispersion/OntoCityGMLNamespace, should merge the TBox soon

Reply:
Done.